blocks/library/list/index.js

```diff
@@ -17,6 +17,47 @@ import { registerBlockType, createBlock } from '../../api';
 import Editable from '../../editable';
 import BlockControls from '../../block-controls';
 
+/**
+ * Converts an array of list items to an array of content.
+ *
+ * @param  {Array} items Array of React list items.
+ * @return {Array}       Array of Editable value arrays.
+ */
+function listItemsToArray( items ) {
+	return items.reduce( ( acc, item ) => {
```

Comment:
I'm not totally certain but it looks like we may be able to use flatMap here and it has some performance advantages according to @aduth test cases. It may also simplify the code :)